src/App.js

```diff
@@ -112,6 +115,12 @@ function App() {
               />
             )}
           />
+          <Route
+            path={'/certificateList'}
+            render={props => (
```

Comment:
Do you need props in the PageCertificatOverview? If not you could just write () instead of props

Reply:
Thanks, you are right. but I need props.history later.